<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineStyleDialog</class>
 <widget class="QDialog" name="LineStyleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>544</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Line Style Editor</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>731</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="scaleGrid">
    <item row="1" column="2">
     <widget class="QLabel" name="scale4">
      <property name="text">
       <string>scale4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="scale6">
      <property name="text">
       <string>scale6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="scale0">
      <property name="text">
       <string>scale0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="scale3">
      <property name="text">
       <string>scale3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="scale1">
      <property name="text">
       <string>scale1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="image4">
      <property name="text">
       <string>image4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="scale5">
      <property name="text">
       <string>scale5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="scale8">
      <property name="text">
       <string>scale8</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="scale2">
      <property name="text">
       <string>scale2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="image3">
      <property name="text">
       <string>image3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="image1">
      <property name="text">
       <string>image1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="image2">
      <property name="text">
       <string>image2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="image0">
      <property name="text">
       <string>image0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="scale7">
      <property name="text">
       <string>scale7</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="image5">
      <property name="text">
       <string>image5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="image6">
      <property name="text">
       <string>image6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="image7">
      <property name="text">
       <string>image7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="image8">
      <property name="text">
       <string>image8</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>471</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="formAlignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="lineColorButton">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Line Color...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineColorText"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineWidthText"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="borderColorButton">
      <property name="text">
       <string>Border Color...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="borderColorText"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="borderWidthText"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="centerLineColorButton">
      <property name="text">
       <string>Center Line Color...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="centerLineColorText"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="centerLineWidthText"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lineWidthLabel">
      <property name="text">
       <string>Line Width</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="borderWidthLabel">
      <property name="text">
       <string>Border Width</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="centerLineWidthLabel">
      <property name="text">
       <string>Center Line Width</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="lineDashLabel">
      <property name="text">
       <string>Dash Array</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineDashText"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="opacityLabel">
      <property name="text">
       <string>Opacity</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="opacityText"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="truncationAtStartLabel">
      <property name="text">
       <string>Truncation at Start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="truncationAtStartText"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="truncationAtEndLabel">
      <property name="text">
       <string>Truncation at End</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="truncationAtEndText"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="distanceLabel">
      <property name="text">
       <string>Highlight Distance</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="distanceText"/>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LineStyleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LineStyleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
